src/adapter/aws-lambda/index.ts

```diff
@@ -1,4 +1,5 @@
 // @denoify-ignore
 export { handle, streamHandle } from './handler'
+export type { APIGatewayProxyResult } from './handler'
```

Comment:
You have to export `APIGatewayProxyResult` from `src/adapter/aws-lambda/handler.ts`.

Reply:
Hi @juho , last one thing!

Reply:
Doh, brain fart. Done